README.md

```diff
@@ -209,6 +210,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * **TCPPort**: Custom TCP port.
 * **RestartService**: If true will restart SQL Service instance service after update. Default false.
 
+###xSQLServerDatabase
+* **Database**: (key) Database to be created or dropped
+* **Ensure**: An enumerated value that describes if Database is to be present or absent.
+* **SQLServer**: The SQL Server for the database
+* **SQLInstance**: The SQL instance for the database 
 ## Versions
 
 ### 1.4.0
```

Comment:
Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)